docs/management/index.md

```diff
@@ -299,20 +299,25 @@ rabbitmqctl set_user_tags full_access administrator
 
 ## Authenticating with OAuth 2 {#oauth2-authentication}
 
-RabbitMQ can be configured to use [JWT-encoded OAuth 2.0 access tokens](https://github.com/rabbitmq/rabbitmq-server/tree/main/deps/rabbitmq_auth_backend_oauth2) to authenticate client applications and management UI users. When doing so, the management UI does
-not automatically redirect users to authenticate against the OAuth 2 server, this must be configured separately. Currently, **Authorization code flow with PKCE** is tested with the following Authorization servers:
+You can configure RabbitMQ to use [JWT-encoded OAuth 2.0 access tokens](https://github.com/rabbitmq/rabbitmq-server/tree/main/deps/rabbitmq_auth_backend_oauth2) to authenticate client applications. But to use OAuth 2.0 authentication in the Management UI, you have to configure it separately. 
 
```

Comment:
Hi @MarcialRosales so we can't start a sentence with "But" so I just run the sentence on like this:

...application, however, to use.......